test/irb/command/test_custom_command.rb

```diff
@@ -123,5 +123,32 @@ def execute(*)
       assert_include(output, "2 FooBar executed")
       assert_include(output, "foobar_description")
     end
+
+    def test_no_meta_command_also_works
+      write_ruby <<~RUBY
+        require "irb"
+        require "irb/command"
```

Comment:
nit: `require "irb"` is not needed for this test

```suggestion
        require "irb/command"
```